EndlessClient/Dialogs/Factories/HelpDialogFactory.cs

```diff
@@ -53,13 +53,11 @@ private string[] GetMessages()
         {
             return new[]
             {
-                _localizedStringFinder.GetString(EOResourceID.ENDLESS_HELP_SUMMARY_1),
```

Comment:
Why were these changes deleted? Can this file be reverted?

Reply:
leftovers on getting used to the repo haha. sorry !